README.md

```diff
@@ -1 +1,18 @@
-WIP - Not ready for use.
+# WIP - Not ready for use.
+
+## Development
+
+Its recommended to setup a virtual environment, e.g. `virtualenv venv`.
+
+Then do `pip install requirements.txt`
+
+You will need the following env vars set. In Prod, these would be passed in via Beep Boop.
+
```

Comment:
More of a reminder... when we point to the Resourcer-dev, we'll want to add instructions on how they fetch these values from the beepboophq site.